fedimint-core/src/util/mod.rs

```diff
@@ -440,8 +440,8 @@ where
                 } else {
                     warn!(
                         target: LOG_CORE,
-                        %error,
-                        %attempts,
+                        ?error,
+                        ?attempts,
```

Comment:
For `error` the `Debug` printing makes sense, less so for `attempts`.

Reply:
Changed attempts back